pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -235,30 +239,35 @@ func (tc *Controller) processJob(key string) error {
 		Preconditions:     &metav1.Preconditions{UID: &fresh.UID},
 	}
 	klog.V(4).Infof("Cleaning up Job %s/%s", namespace, name)
-	return tc.client.BatchV1().Jobs(fresh.Namespace).Delete(context.TODO(), fresh.Name, options)
+	if err := tc.client.BatchV1().Jobs(fresh.Namespace).Delete(context.TODO(), fresh.Name, options); err != nil {
+		return err
+	}
+	metrics.TimeToDeletionSeconds.Observe(timeToDeletion.Seconds())
```

Comment:
shouldn't this metric include the time it took to call the API?

Reply:
make sense.